backend/src/services/optimized_guide_service.py

```
@@ -142,19 +156,28 @@ async def generate_optimized_guide(
             if progress_callback:
-                await progress_callback(100, f"Guide ready! Generated in {generation_time:.1f}s")
+                await progress_callback(
+                    100, f"Guide ready! Generated in {generation_time:.1f}s"
+                )
             
-            logger.info(f"Guide generated successfully for {destination} in {generation_time:.1f}s")
+            logger.info(
+                f"Guide generated successfully for {destination} in "
+                f"{generation_time:.1f}s"
+            )
             return complete_guide
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)